src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright © 2017 - 2018 Chocolatey Software, Inc

```

Comment:
I guess not specifically related to this PR, but should we look at updating the license / copyright dates for this repo at some point? 🤔

Should this have a newer date since it's a newer file? Not sure what the process is for these tbqh.

Reply:
I actually had the same thought about this as I was adding it in....

I have created an issue to track getting this done ahead of the next release: https://github.com/chocolatey/choco/issues/2261 @ferventcoder has a method for bulk updating these, so will get with him about how that is done.